src/app/components/resources/ResourceRow.tsx

```diff
@@ -13,29 +40,75 @@ const useStyles = makeStyles((theme: Theme) =>
 );
 
 type ResourceRowProps = {
-    title: React.ReactElement;
+    demoUrl?: string;
+    name?: string;
+    package?: string;
     description: string;
-    repository: string;
     divider?: boolean;
-    rightComponent?: JSX.Element;
+    repository: string;
+    bugLabels?: string[];
 };
-export const ResourceRow: React.FC<ResourceRowProps> = (props): JSX.Element | null => {
+export const ResourceRow: React.FC<ResourceRowProps> = (props): JSX.Element => {
+    const { name, demoUrl, repository, description, divider, bugLabels, package: packageName } = props;
+    const theme = useTheme();
+
+    const [version, setVersion] = useState<string>();
+    const repositoryLink = `https://github.com/pxblue/${repository}`;
     const small = useMediaQuery('(max-width:799px)');
+    const xs = useMediaQuery('(max-width:499px)');
     const classes = useStyles();
-    const theme = useTheme();
 
-    const repositoryLink = `https://github.com/pxblue/${props.repository}`;
+    // Make the API calls for the live information
+    useEffect(() => {
+        let isMounted = true;
+        if (packageName) {
+            const loadVersion = async (): Promise<void> => {
+                const npmVersion = await getNpmVersion(packageName);
+                if (isMounted) {
+                    setVersion(npmVersion);
+                }
+            };
+            loadVersion();
+            return (): void => {
+                isMounted = false;
+            };
+        }
+    }, [repository, bugLabels, packageName]);
+
+    const buttons = useCallback(
+        (): JSX.Element => (
+            <ButtonRow isPackage={Boolean(packageName)} small={small} repository={repository} demoUrl={demoUrl} />
+        ),
+        [small, repository, demoUrl]
+    );
 
     return (
         <div>
             <InfoListItem
                 hidePadding
                 style={{ paddingRight: theme.spacing(1) }}
-                divider={!small && props.divider ? 'full' : undefined}
-                title={props.title}
-                subtitle={props.description}
+                divider={!small && divider ? 'full' : undefined}
+                title={
+                    <div className={classes.flex} style={{ width: small ? '100%' : 'auto' }}>
+                        <Typography className={classes.title} noWrap>
+                            {packageName && xs ? packageName.replace('@pxblue/', '') : packageName}
+                            {name}
+                        </Typography>
+                        {version && (
+                            <Typography
+                                variant={'subtitle2'}
+                                className={classes.version}
+                                onClick={(): void => {
+                                    window.open(`https://www.npmjs.com/package/${packageName}`, '_blank');
+                                }}
+                            >{`@${version}`}</Typography>
```

Comment:
This click lacks some discoverability... fine with it for now, but I'd prob refine the design (like add an npm icon on the right)